sklearn/manifold/spectral_embedding_.py

```diff
@@ -384,7 +388,11 @@ class SpectralEmbedding(BaseEstimator):
       Jianbo Shi, Jitendra Malik
       http://citeseer.ist.psu.edu/viewdoc/summary?doi=10.1.1.160.2324
     """
-
+    """
+    @deprecated("Class 'SpectralEmbedding' has been renamed to"
+                "'LaplacianEigenmaps' and will be removed"
+                "in release 0.19")
+    """
```

Comment:
and this line and move the above 3 lines before the class declaration for the Class to be deprecated upon init :) see [this](https://github.com/scikit-learn/scikit-learn/blob/0c366942da4263301534a425bf9611b8048acab2/sklearn/gaussian_process/gaussian_process.py#L61) for a reference...